src/features/visiting/hud/components/InventoryTabContent.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useEffect, useState } from "react";
+import { Box } from "components/ui/Box";
+import { OuterPanel } from "components/ui/Panel";
+import { ITEM_DETAILS } from "features/game/types/images";
+import { InventoryItemName } from "features/game/types/game";
+
+import { SEEDS, CropName } from "features/game/types/crops";
+
+import timer from "assets/icons/timer.png";
+import lightning from "assets/icons/lightning.png";
+
+import { secondsToMidString } from "lib/utils/time";
+import classNames from "classnames";
+import { useShowScrollbar } from "lib/utils/hooks/useShowScrollbar";
+import { useScrollIntoView } from "lib/utils/hooks/useScrollIntoView";
+import { Inventory, TabItems } from "./InventoryItems";
+import { hasBoost } from "features/game/lib/boosts";
+import { getCropTime } from "features/game/events/plant";
+import { getKeys } from "features/game/types/craftables";
+
+const ITEM_CARD_MIN_HEIGHT = "148px";
+
+interface Props {
+  tabItems: TabItems;
+  selectedItem?: InventoryItemName;
+  setDefaultSelectedItem: (item: InventoryItemName) => void;
+  inventory: Inventory;
+  inventoryItems: InventoryItemName[];
+  onClick: (item: InventoryItemName) => void;
+}
+
+const TAB_CONTENT_HEIGHT = 384;
+
+const isSeed = (selectedItem: InventoryItemName) => selectedItem in SEEDS();
+
+export const InventoryTabContent = ({
```

Comment:
Do we need to duplicate this component? It doesn't look like it is specifically reading from the `gameMachine`